components/CategoriesList/CategoriesList.tsx

```diff
@@ -52,7 +52,7 @@ function CategoriesList({ categories, showAllCategories = false, isStatic }: Pro
     }, [categories, showExtraCategories, currentLocale]);
 
     return (
-        <>
+        <div>
```

Comment:
I see that the problem was with the `:last-child` selector.

But, adding this `div` effectively disabled the styles applied from the parent containers in story cards.
It would make sense to move the base layout styles from wrapping container in the story cards to this div, and add a `className` prop to the component, so that we can add extra styles.

So, for `HighlightedStoryCard`:
- Remove the `categoriesWrapper` div and its class
- The font-size and line-height properties are redundant and can be removed
- The flex properties should move to the `CategoriesList` styles

For `StoryCard`:
- Remove the `categories` div
- Apply the `categories` from the component stylesheet as `className` prop for Categories list
- Remove flex properties from `categories` class name.

Reply:
Applied these changes